packages/strapi-admin/services/action.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const { isNil } = require('lodash/fp');
+const { getService } = require('../utils');
+const { AUTHOR_CODE, PUBLISH_ACTION } = require('./constants');
+
+// TODO: move actionProvider here instead of in the permission service
+
+/**
+ * Returns actions available for a role.
+ * @param {object} roleId
+ * @returns {object[]}
+ */
+const getActionsByRoleId = async roleId => {
```

Comment:
Can we rename it to: `getAllowedActionsFor` or `getAllowedActionsForRole` ? As actions don't belong to a role, the naming confuses me

Reply:
Actually they do with this management rule :D but sounds good